server/sonar-db-dao/src/test/java/org/sonar/db/component/SnapshotDtoTest.java

```diff
@@ -48,4 +55,17 @@ public void test_getter_and_setter() throws Exception {
     assertThat(snapshotDto.getPeriodDate()).isEqualTo(parseDate("2014-06-01").getTime());
   }
 
+  @Test
+  public void fail_if_version_name_is_longer_then_100_characters() throws Exception {
+    SnapshotDto snapshotDto = new SnapshotDto();
+    snapshotDto.setVersion(null);
+    snapshotDto.setVersion("1.0");
+    snapshotDto.setVersion(IntStream.range(0, 100).mapToObj(i -> "a").collect(Collectors.joining()));
```

Comment:
Apache provides utility methods, `StringUtils.repeat(…)`. I think it's overkill to use stream here to build a string

Reply:
Done.